scripts/tracecontext-integration-test.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+# set -e 
+# hard-coding the git tag to ensure stable builds.
+TRACECONTEXT_GIT_TAG="98f210efd89c63593dce90e2bae0a1bdcb986f51"
+# clone w3c tracecontext tests
+mkdir -p target
+rm -rf ./target/trace-context
+git clone https://github.com/w3c/trace-context ./target/trace-context
```

Comment:
Out of curiosity: why do it this way instead of adding as a submodule?

Reply:
I think submodules work fine here, but generally track a branch vs a commit. I'll look through and may re-add as a followup, or fix it here depending on approvers.